scripts/python/met/dataplane.py

```diff
@@ -1,16 +1,62 @@
 import os
+import sys
 import numpy as np
 import netCDF4 as nc
+import xarray as xr
+
+from importlib import util as import_util
+from met.logger import logger
 
 ###########################################
 
-class dataplane():
+class dataplane(logger):
+
+   KEEP_XARRAY = True
+   class_name = "dataplane"
+
+   MET_FILL_VALUE = -9999.
+   ATTR_USER_FILL_VALUE = 'user_fill_value'
+
+   @staticmethod
+   def call_python(argv):
+       logger.log_msg(f"Python Script:\t{repr(argv[0])}")
```

Comment:
Is there a better statement than "Python Script" because sometimes this reports `met.dataplane`, which is the class and not a script, i.e.:

```
DEBUG 3: Initializing MET compile time python instance: /usr/local/met-python3/bin/python3.8
 PYTHON: Python Script:	'met.dataplane'
```

Reply:
How about "Python module"?

Reply:
That's good with me, thanks!